solution/operator/service_account.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: zenko-operator
```

Comment:
Not needed either, a `ServiceAccount` named `{{ solution.name }}-operator` will be created by MetalK8s.
In this case, this account would be named `zenko-operator`.

Reply:
Removed the file